src/main/java/com/worldwarofants/game/module/newgame/NewGameCommandHandler.java

```diff
@@ -0,0 +1,30 @@
+package com.worldwarofants.game.module.newgame;
+
+import java.util.Map;
+
+import com.worldwarofants.game.arch.console.command.AbstractCommandHandler;
+import com.worldwarofants.game.arch.console.command.ICommand;
+import com.worldwarofants.game.module.ModuleName;
+
+public class NewGameCommandHandler extends AbstractCommandHandler<NewGameController> {
+	
+	public static final String COMMAND_SHOW_NEW_GAME_SCREEN = "showNewGameScreen";
```

Comment:
That command is still very long.

Reply:
So the starting commands that get called first on navigating to a module I just left since really people wouldn't be typing those. I can still make it shorter though it won't matter.

Reply:
I see your point but we still can't guarantee that it's gonna work that way and that people won't be typing them.